src/components/internal/convert.ts

```diff
@@ -0,0 +1,123 @@
+import {html} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import {InternalElement} from './InternalElement';
+import {fiat} from '@getalby/lightning-tools';
+import {withTwind} from '../twind/withTwind.js';
+import {classes} from '../css/classes.js';
+
+@customElement('satoshi-converter')
+export class SatoshiConverter extends withTwind()(InternalElement) {
```

Comment:
I think we should have just two components:
- `currency-switcher` which when clicked toggles the dropdown and handles picking the currency
- existing `bc-balance` which renders the balance in whatever currency the user has chosen, and only does fiat conversion if currency is not BTC